commit 8837295a73f3500b32e18f9862c7bdde0b958648
Author: Eduardo Valentin <eduardo.valentin@ti.com>
Date:   Tue Mar 26 21:38:34 2013 +0000

    thermal: add a warning for temperature emulation feature
    
    Because this feature is for debuging purposes, it is highly
    recommended to do not enable this on production systems.
    This patch adds warnings for system integrators, so that
    people are aware of this potential security issue.
    
    Signed-off-by: Eduardo Valentin <eduardo.valentin@ti.com>
    Signed-off-by: Zhang Rui <rui.zhang@intel.com>

diff --git a/Documentation/thermal/sysfs-api.txt b/Documentation/thermal/sysfs-api.txt
index 6859661c9d31..277530a5786c 100644
--- a/Documentation/thermal/sysfs-api.txt
+++ b/Documentation/thermal/sysfs-api.txt
@@ -265,6 +265,10 @@ emul_temp
 	Unit: millidegree Celsius
 	WO, Optional
 
+	  WARNING: Be careful while enabling this option on production systems,
+	  because userland can easily disable the thermal policy by simply
+	  flooding this sysfs node with low temperature values.
+
 *****************************
 * Cooling device attributes *
 *****************************